web-common/src/features/dashboards/time-series/MetricsTimeSeriesCharts.svelte

```diff
@@ -241,9 +250,32 @@
     class="bg-white sticky left-0 top-0 overflow-visible"
     style="z-index:101"
   >
-    <div style:height="20px" style:padding-left="24px" />
     <!-- top axis element -->
     <div />
+    <div bind:this={axisTop} style:height="24px" style:padding-left="24px">
```

Comment:
You could consider componentizing this element. Maybe extreme, but an option would be to create a subfeature `features/dashboards/time-series/zoom`, so we can colocate the code & make this file smaller.